src/lib/controller/store.ts

```diff
@@ -134,7 +135,7 @@ export class StoreController {
 
     // Debounce the update to 1 second to avoid multiple rapid calls
     clearTimeout(this.#sendDebounce);
-    this.#sendDebounce = setTimeout(debounced, this.#onReady ? 0 : debounceBackoff);
+    this.#sendDebounce = setTimeout(debounced, this.#onReady ? 0 : debounceBackoffReceive);
```

Comment:
`this.#sendDebounce` --> `debounceBackoffReceive`..?  Zat right?

Reply:
yep! it is addressed!

Reply:
thanks for the callout